taichi/codegen/llvm/codegen_llvm.h

```diff
@@ -63,14 +63,19 @@ class TaskCodeGenLLVM : public IRVisitor, public LLVMModuleBuilder {
   std::unordered_set<int> struct_for_tls_sizes;
   const Callable *current_callable{nullptr};
 
+  // The task_codegen_id and task_counter are used to generate unique task ids
+  int task_codegen_id{0};
```

Comment:
nit: from the logic above it seems `task_codegen_id` represents offload task id in a kernel right? would you mind explaining these two in comments?

Reply:
> from the logic above it seems task_codegen_id represents offload task id in a kernel right?

Yes!

> would you mind explaining these two in comments?

I added more detailed comment.